src/coreclr/tools/StressLogAnalyzer/StressLogPlugin.cpp

```diff
@@ -1397,11 +1400,11 @@ int ProcessStressLog(void* baseAddress, int argc, char* argv[])
 
     CorClrData corClrData(hdr);
     FILE* outputFile = stdout;
-    if (s_outputFileName != nullptr)
+    if (s_outputFileNameW != nullptr)
     {
-        if (_wfopen_s(&outputFile, s_outputFileName, W("w")) != 0)
+        if (_wfopen_s(&outputFile, s_outputFileNameW, W("w")) != 0)
```

Comment:
Can't we just use fopen_s now that we have the 8 bit char version of the filename?

Reply:
Good question. Let me see. I'd prefer that anyways.